### ISSUE TITLE
Can't be built with MonoDevelop on MacOS('Silverlight 4.0 Client Profile' not installed)

### ISSUE THREAD

**Issue by yhslai** [jsonfx/jsonfx#27]
I open the project, and press Build, but what I got is:

```
Error: Framework 'Silverlight 4.0 Client Profile' not installed. (JsonFx)
```

I don't know why it needs Silverlight to build. Any idea?

MonoDevelop version: 2.8.2
OSX version: 10.8

**Comment by increddibelly** [comment]
I agree with raincole - I find it a little awkward to have to install the silverlight dev tools (which *lol* makes a desperate attempt to push Bing) just to open the project. Is this really necessary? dare I say it - who uses silverlight anymore?
